### ISSUE TITLE
[tools-v2]:add bs stop copyset snapshot

### ISSUE THREAD

**Issue by Cyber-SiKu** [opencurve/curve#2579]
**Is your feature request related to a problem? (你需要的功能是否与某个问题有关?)**

we'd like to support `bs delete copyset snapshot` command  in [curve tool](https://github.com/opencurve/curve/tree/master/tools-v2)

**Describe the solution you'd like (描述你期望的解决方法)**
- The implementation of the old tool is here：
- 
https://github.com/opencurve/curve/blob/44949da50bacd046e2143448fb524ca1dece908b/tools/snaptool/snaptool.py#L22-L23

**Describe alternatives you've considered (描述你想到的折衷方案)**

**Additional context/screenshots (更多上下文/截图)**

**Comment by ZackSoul** [comment]
![issue](https://github.com/opencurve/curve/assets/91828262/14acd21a-2e21-4325-82f4-8cd0a000c900)
Do I need to implement these two functions from [curtools.py](http://curtools.py/) using Go, or are there already corresponding functions implemented in Go?

**Comment by Cyber-SiKu** [comment]
> ![issue](https://user-images.githubusercontent.com/91828262/261265907-14acd21a-2e21-4325-82f4-8cd0a000c900.png) Do I need to implement these two functions from [curtools.py](http://curtools.py/) using Go, or are there already corresponding functions implemented in Go?Also, which parameter in Go corresponds to the UUID in the arguments?

There should be no need for you to implement this aspect. For uuid, you can look at the implementation of the original python code and implement the corresponding parameters yourself.

**Comment by Cyber-SiKu** [comment]
> Hi, I have read the source code of the python implementation and implemented the main framework of the code, but I don't find the relevant rpc for canceling the snapshot as shown in the picture, what should I do? ![interface](https://user-images.githubusercontent.com/91828262/265475900-304b5e1f-d37f-4bb0-bc84-6ba20a5d05a5.png)

[Read the source code](https://user-images.githubusercontent.com/91828262/261265907-14acd21a-2e21-4325-82f4-8cd0a000c900.png), you can see that this should send an http request instead of an rpc Request, follow this to implement a go version.

**Comment by ZackSoul** [comment]
Yes, I know that the python implementation does this by sending the http request, but I didn't find the rpc about this command in the go language version, does this mean that I need to implement the relevant rpc command myself.

**Comment by Cyber-SiKu** [comment]
> Yes, I know that the python implementation does this by sending the http request, but I didn't find the rpc about this command in the go language version, does this mean that I need to implement the relevant rpc command myself.

[Here](https://github.com/opencurve/curve/blob/e822715d62d6fe6f65555be1c81aed2cbcba18aa/tools-v2/pkg/cli/command/base.go#L232-L265) Send http request, if it is not suitable, you can add it one.

**Comment by ZackSoul** [comment]
I want to assign this issue

**Comment by CrystalAnalyst** [comment]
Hi, I'm a junior major in computer science in HangzhouDianzi University currently. I am familiar with programming languages like Golang and C++. (Also, I'm also a newbie to Rust). I'm interested in WebFramework Development, Cloud-Native, Distributed System and Storage System. I'm good at Golang development and familiar with the go stand library. I wish I can take on this issue to pracitce my skills and capability further. Please assign it to me.

**Comment by Subzeroo7** [comment]
> Hi, I'm a junior major in computer science in HangzhouDianzi University currently. I am familiar with programming languages like Golang and C++. (Also, I'm a newbie to Rust). I'm interested in WebFramework Development, Cloud-Native, Distributed System and Storage System. I'm good at Golang development and familiar with the go stand library. I wish I can take on this issue to pracitce my skills and capability further. Please assign it to me.

Sorry，this issue has been assigned，please choose others.
